# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../pynumarray/compile-fix.patch
# Copyright (C) 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

# this patch avoids lock up on compile, strange ir that it's ok compiling
# it manually, if you think that's wrong please fix me :) Baldzius
diff -Nur numarray-1.5.0-orig/setup.py numarray-1.5.0/setup.py
--- numarray-1.5.0-orig/setup.py	2005-11-25 16:02:55.000000000 +0000
+++ numarray-1.5.0/setup.py	2005-12-23 16:51:30.000000000 +0000
@@ -227,6 +227,6 @@
 # Do the installation.
 #
 if __name__ == "__main__":
-  if "config" not in sys.argv and ("build" in sys.argv or "install" in sys.argv):
-    sys.argv.insert(sys.argv.index("setup.py")+1, "config")
+#  if "config" not in sys.argv and ("build" in sys.argv or "install" in sys.argv):
+#    sys.argv.insert(sys.argv.index("setup.py")+1, "config")
   main()
